src/main/java/App.java

```diff
@@ -63,11 +66,17 @@ public static void main(String[] args) throws Exception {
       System.out.println(useEquivalence);
       System.out.println(useSubsumption);
 
-      Object result = manager.handle(sourceFileLocation, targetFileLocation, useEquivalence, useSubsumption);
-
-
-      ctx.result(result.toString());
+      URIAlignment result = manager.handle(sourceFileLocation, targetFileLocation, useEquivalence, useSubsumption);
 
+      String json = "[";
+      for (Cell cell: result.getArrayElements()) {
+        json += String.format("{\"source\": \"%s\", \"target\": \"%s\", \"relation\": \"%s\", \"confidence\": %s},"
+          , CellParser.getSource(cell), CellParser.getTarget(cell), CellParser.getRelation(cell), CellParser.getConfidence(cell));
```

Comment:
This is where the magic happens, but once I run this from a POST-request I see something interesting! 

```json
[
    {
        "source": "Misc",
        "target": "Unpublished",
        "relation": "=",
        "confidence": 0,
        00
    }
]
```
What's happening to the confidence-property is that it's set to 0.00, and then formatted (in `CellParser.getConfidence`) in **Norwegian!** This is kinda bad, but it does it to be nice. To change this, we need to change the String-format locale. 

Two solutions:

1. Solve it only here, by replacing
```java
public static String getConfidence(Cell cell){
    return String.format("%.2f", cell.getStrength());  
}
```
with

```java
// import java.util.Locale;

public static String getConfidence(Cell cell){
    return String.format(Locale.ENGLISH, "%.2f", cell.getStrength());  
}
```

2. Solve it globally by setting the global locale. This is done by adding this line to the start of our program (inn App.java):
```java 
Locale.setDefault(Locale.ENGLISH);
```

I vote for solution 2, but no matter the choice - it should be fixed before we merge this.